kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JDBCMetadataStore.scala

```diff
@@ -257,14 +257,15 @@ class JDBCMetadataStore(conf: KyuubiConf) extends MetadataStore with Logging {
       filter: MetadataFilter,
       from: Int,
       size: Int,
-      desc: Boolean = false): Seq[Metadata] = {
+      desc: Boolean = false,
+      orderByKeyId: Boolean = true): Seq[Metadata] = {
```

Comment:
the parameters are evolving to Spark's `SortOrder`

```
case class SortOrder(
    child: Expression,
    direction: SortDirection,
    nullOrdering: NullOrdering,
    sameOrderExpressions: Seq[Expression])
```

how about using some SQL terminologies to redesign the method's signature?
```
override def getMetadataList(
      filter: MetadataFilter,
      offset: Int,
      limit: Int,
      orderBy: Option[String],
      direction: String = "ASC"): Seq[Metadata]
```